File: PicStory-Api/PicStory.Core/DTOs/AlbumDTO.cs
﻿using PicStory.CORE.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PicStory.CORE.DTOs
{
    public class AlbumDTO
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime CreatedAt { get; set; }

        public User User { get; set; }
        public List<Photo> Photos { get; set; }
        public List<SharedAlbum> SharedAlbums { get; set; }
    }
}

File: PicStory-react/src/models/AlbumDTO.ts
export type AlbumDTO = {
  id: number;
  userId:number;
  name: string;
  description: string;
  createdAt: string;
  updatedAt: string;
  photos: string[];  // נניח שזה מערך של תמונות (לפי הצורך תוכל לעדכן את סוג המידע)
  sharedAlbums: string[];
}
